roles/backpack_kube/tasks/main.yml

```diff
@@ -1,48 +1,79 @@
 ---
+- name: Set facts
+  set_fact:
+    operator_namespace: backpack
```

Comment:
Why are we setting facts here? why aren't these in the defaults ?

Reply:
set a few facts to default('xxx') in case the end user wants to override them to something else later.